AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSPaginatedDocumentsInternal.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSPage.h"
+#import "MSReadOptions.h"
+
+@interface MSPaginatedDocuments()
+
+// Read-only.
+@property(nonatomic, copy, readonly) NSString *partition;
+@property(nonatomic, readonly) Class documentType;
+@property(nonatomic, readonly) MSReadOptions *readOptions;
+
+// Read-write (to implement pagination).
+@property(nonatomic, readwrite) MSPage *currentPage;
```

Comment:
`readwrite` is by default. You can remove that.
```suggestion
@property(nonatomic) MSPage *currentPage;
```